Resources/Private/JavaScript/Host/Redux/Sagas/UI/Inspector/index.js

```diff
@@ -0,0 +1,29 @@
+import {takeEvery} from 'redux-saga';
+import {put} from 'redux-saga/effects';
+import {$get} from 'plow-js';
+
+import {actionTypes, actions} from 'Host/Redux/';
+
+const getTransientInspectorValues = $get(['ui', 'inspector', 'valuesByNodePath']);
+
+export function* applyInspectorState(getState) {
+    yield* takeEvery(actionTypes.UI.Inspector.APPLY, function* applyAllChanges(action) {
+        const state = getState();
+        const {nodeContextPath} = action.payload;
+
+        const transientNodeInspectorValues = $get([nodeContextPath], getTransientInspectorValues(state));
+
+        for (const key of Object.keys(transientNodeInspectorValues)) {
+            yield put(actions.Changes.add({
```

Comment:
Besides, there's no `actions.Changes.add` anymore, I renamed it to `persistChange`. I would add another action, e.g. `persistChanges` or something, to take an array of changes. Our API takes arrays of changes in any case, so it should be very easy.